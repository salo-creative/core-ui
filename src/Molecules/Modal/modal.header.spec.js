import React from 'react';
import 'jest-styled-components';

import { renderWithTheme } from '../../../test';

// COMPONENT
import ModalHeader from './modal.header';

test('Renders against snapshot', async () => {
  const { container } = renderWithTheme(
    <ModalHeader />
  );
  expect(container.firstChild).toMatchSnapshot();
});