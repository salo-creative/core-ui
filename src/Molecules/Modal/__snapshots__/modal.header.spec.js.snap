// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders against snapshot 1`] = `
.c3 {
  height: 14px;
  width: 14px;
  margin: 0;
}

.c0 {
  position: relative;
  position: -webkit-sticky;
  position: sticky;
  top: 0;
  left: 0;
  z-index: 10;
  width: 100%;
  background: #fff;
  border-bottom: 1px solid #EAEAEA;
  text-align: center;
  padding: 0 2rem;
  height: 4.5rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c2 {
  background: none;
  border: none;
  cursor: pointer;
  padding: 1rem;
  position: absolute;
  right: 1rem;
  opacity: 0.7;
  line-height: 0;
  top: 50%;
  -webkit-transform: translateY(-50%);
  -ms-transform: translateY(-50%);
  transform: translateY(-50%);
  z-index: 1;
}

.c2:hover {
  opacity: 1;
}

.c1 {
  margin: 0;
  font-weight: 300;
  font-size: 1.6rem;
  -webkit-letter-spacing: 1px;
  -moz-letter-spacing: 1px;
  -ms-letter-spacing: 1px;
  letter-spacing: 1px;
}

<div
  class="c0 salo-modal__header"
>
  <h5
    class="c1"
  >
    Modal title
  </h5>
  <button
    class="c2 salo-modal__close"
  >
    <svg
      class="c3"
      size="14"
      viewBox="0 0 24 24"
    >
      <path
        d="M19,6.41L17.59,5L12,10.59L6.41,5L5,6.41L10.59,12L5,17.59L6.41,19L12,13.41L17.59,19L19,17.59L13.41,12L19,6.41Z"
        fill="#00222b"
      />
    </svg>
  </button>
</div>
`;
